art/attacks/evasion/projected_gradient_descent/projected_gradient_descent_numpy.py

```diff
@@ -152,37 +162,15 @@ def _set_targets(self, x: np.ndarray, y: np.ndarray, classifier_mixin: bool = Tr
 
         return targets
 
-    @staticmethod
-    def _get_mask(x: np.ndarray, classifier_mixin: bool = True, **kwargs) -> np.ndarray:
-        """
-        Get the mask from the kwargs.
-
-        :param x: An array with the original inputs.
-        :param classifier_mixin: Whether the estimator is of type `ClassifierMixin`.
-        :param mask: An array with a mask to be applied to the adversarial perturbations. Shape needs to be
-                     broadcastable to the shape of x. Any features for which the mask is zero will not be adversarially
-                     perturbed.
-        :type mask: `np.ndarray`
-        :return: The mask.
-        """
-        mask = kwargs.get("mask")
-
-        if mask is not None:
-            if classifier_mixin:
-                # Ensure the mask is broadcastable
-                if len(mask.shape) > len(x.shape) or mask.shape != x.shape[-len(mask.shape) :]:
-                    raise ValueError("Mask shape must be broadcastable to input shape.")
-
-            else:
-                raise ValueError("Mask is only supported for classification.")
-
-        return mask
-
     def _check_params(self) -> None:
         super(ProjectedGradientDescentCommon, self)._check_params()
 
-        if self.eps_step > self.eps:
-            raise ValueError("The iteration step `eps_step` has to be smaller than the total attack `eps`.")
+        if isinstance(self.eps, float):
+            if self.eps_step > self.eps:
+                raise ValueError("The iteration step `eps_step` has to be smaller than the total attack `eps`.")
```

Comment:
This might only be true for `norm=np.inf`.

Reply:
solved.